CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.0.0 (02/12/2019)
+
+**Common**
+
+* Project structure was changed, public API aligned to the HERE OLP SDK for C++.
+* Unit tests was improved
+
+**olp-sdk-dataservice-read**
+
+* Added new class `OlpClientSetting` with `KeyValueCache` instance to be used for context
+* Added new class `VersionedLayerClient` that is used to access versioned layers in OLP. Class implements `GetData`, `GetPartitions` methods.
+* Added new class `VolatileLayerClient` that is used to access volatile layers in OLP. Class implements `GetPartitions`, `GetData` methods.
+* Added new class `ArtifactClient` that is used to access artifact service in OLP. Class implements `getSchemaDetails`, `getSchema` methods.
+* Added new class `CatalogClient` that is used to access catalog metada in OLP. Class implements `getCatalog`, `getLatestVersion`, `getVersions` methods.
```

Comment:
`CatalogClient` was there before you just modified it, right? Please write what dropped and what changed.

Reply:
```suggestion
* Added the `CatalogClient` class that is used to access catalog metada on OLP. This class implements the `getCatalog`, `getLatestVersion`, and `getVersions` methods.
```

Reply:
updated